xscen/aggregate.py

```diff
@@ -171,34 +390,87 @@ def climatological_mean(
                 )
             ]
         else:
-            raise ValueError("The type of 'time' could not be understood.")
+            raise ValueError("The type of 'time' was not understood.")
         ds_rolling = ds_rolling.drop_vars({"month", "year", "time", "day"})
         ds_rolling = ds_rolling.assign_coords(time=time_coord).transpose("time", ...)
 
+        # append to list of results
         concats.extend([ds_rolling])
+        # end loop over periods
+
+    # concatenate results
     ds_rolling = xr.concat(concats, dim="time", data_vars="minimal")
 
-    # modify attrs and history
+    # update data_vars names, attrs, history
+    if rename_variables:
+        ds_rolling = ds_rolling.rename_vars(
+            {vv: f"{vv}_clim_{op}" for vv in ds_rolling.data_vars}
```

Comment:
I'm not a fan of `_clim_{op},` but I don't really have any better idea... Suggestions?

Reply:
I am open to suggestions, I'll keep this first shot for now.
Aggregation rather than climatological operation?